chart/flux/templates/helm-operator-deployment.yaml

```diff
@@ -126,6 +126,7 @@ spec:
         - --charts-sync-interval={{ .Values.helmOperator.chartsSyncInterval }}
         - --update-chart-deps={{ .Values.helmOperator.updateChartDeps }}
         - --log-release-diffs={{ .Values.helmOperator.logReleaseDiffs }}
+        - --workers={{ .Values.helmOperator.workers }}
```

Comment:
Please use an `if` here to make the change backwards compatible, otherwise if someone uses this chart with an older Flux version it will crash at startup.

Reply:
Turbo patched.